pkg/cmd/roachtest/run.go

```diff
@@ -118,6 +132,32 @@ func runTests(register func(registry.Registry), filter *registry.TestFilter) err
 		opt.debugMode = NoDebug
 	}
 
+	if roachtestflags.SideEyeApiToken != "" {
```

Comment:
For CI, we'll need to update the wrapper scripts to thread the token. E.g., see [1], which threads the value to `--use-spot`.

[1] https://github.com/cockroachdb/cockroach/pull/124258